contracts/protocol/libraries/logics/LendingMarketUserLogic.sol

```diff
@@ -434,4 +435,13 @@ library LendingMarketUserLogic {
                 );
         }
     }
+
+    function _isCovered(address _user, bytes32 _ccy) internal view {
+        (bool isEnoughCollateral, bool isEnoughDepositInOrderCcy) = AddressResolverLib
+            .tokenVault()
+            .isCovered(_user, _ccy);
+
```

Comment:
@biga816 
Just a question. Do you see any case in which both `isEnoughDepositInOrderCcy` and `isEnoughCollateral ` are false? Then, if that's the case, do we need an event like `NotEnoughDepositAndCollateral`?

Reply:
It can happen, but `isEnoughDepositInOrderCcy` can be false only when the user tries to execute invalid lending orders. In this case, `NotEnoughDeposit` is enough to understand what is happening.